File: Odd or Even.c
// Find a number whether it is odd or even. Zero is neither odd or even

#include<stdio.h>
int getNumberFromUser(){
    int number;
    scanf("%d",& number);
    return number;
}

void findOddOrEven(int a){
    if(a%2!=0){
        printf("this is odd nuumber ");
    }else{
    printf("this is even number ");    
    }
    }
    void main(){
        printf("enter the number ");
        int a= getNumberFromUser();
        findOddOrEven(a);
    }

File: control flow problems/Odd or Even.js
// Find a number whether it is odd or even. Zero is neither odd or even

const prompt = require("prompt-sync")();

const a = parseInt(prompt("Enter the number = "));

if(a % 2 != 0 ){
    console.log(" Odd ");

}else if(a % 2 == 0 ){
    console.log(" Even ");
}
